test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -32,18 +32,48 @@
 #include <iostream>
 #include <map>
 
+#include "connection_simulator.h"
 #include "error_simulator.h"
 #include "timestamp_manager.h"
 
-session_simulator::session_simulator() : _has_commit_ts(false), _txn_running(false) {}
+session_simulator::session_simulator()
```

Comment:
It is usually good practice to init this list in the same order as those variables were declared in the header file:
```suggestion
session_simulator::session_simulator(): _has_commit_ts(false), _txn_running(false),...
```